modules/custom/az_event/az_event_trellis/src/Plugin/QueueWorker/AZDeferredMedia.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Drupal\az_event_trellis\Plugin\QueueWorker;
+
+use Drupal\Component\Utility\Crypt;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+use Drupal\media\Entity\Media;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Queue worker to lazily download media from integrations.
+ *
+ * @QueueWorker(
+ *   id = "az_deferred_media",
+ *   title = @Translation("Quickstart Deferred Media"),
+ *   cron = {
+ *     "time" = 30,
+ *   },
+ * )
```

Comment:
```suggestion
#[QueueWorker(
  id: 'az_deferred_media',
  title: new TranslatableMarkup('Quickstart Deferred Media'),
  cron: ['time' => 30]
)]
```

Looks like QueueWorker plugins have been converted to attributes as of 10.3.0: https://www.drupal.org/node/3229001